examples/lighting-app/qpg/BUILD.gn

```diff
@@ -1,16 +1,16 @@
-# Copyright (c) 2020 Project CHIP Authors
+#Copyright(c) 2020 Project CHIP Authors
 #
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
+#Licensed under the Apache License, Version 2.0(the "License");
+#you may not use this file except in compliance with the License.
+#You may obtain a copy of the License at
 #
-# http://www.apache.org/licenses/LICENSE-2.0
+#http: // www.apache.org/licenses/LICENSE-2.0
```

Comment:
these edits seem off - url is changed and the extra whitespace helped readability. please undo (I imagine this was some automated script)